src/grt/src/GlobalRouter.cpp

```diff
@@ -828,9 +828,12 @@ bool GlobalRouter::makeFastrouteNet(Net* net)
                                        max_layer - 1,
                                        net->getSlack(),
                                        edge_cost_per_layer);
+    int min_pin_layer = std::numeric_limits<int>::max();
```

Comment:
@eder-matheus Would you mind throwing in a comment with a link to the various GitHub issues and discussion on why this particular code is here.

I'd like to make sure we don't lose the context if we want to change it in the future.